lib/s3_cors_rulesets.rb

```diff
@@ -10,8 +12,55 @@ class S3CorsRulesets
 
   BACKUP_DIRECT_UPLOAD = {
     allowed_headers: ["*"],
-    allowed_methods: ["PUT"],
+    expose_headers: ["ETag"],
+    allowed_methods: ["GET", "HEAD", "PUT"],
```

Comment:
Would like to clarify why we need `GET` and `HEAD` as compared to just `PUT` previously.

Reply:
Will discuss this internally as well, but it is because this will be needed for direct S3 backup multipart uploads, and I would prefer to apply all these new rules at the same time.